simtools/configuration/configurator.py

```diff
@@ -277,6 +277,10 @@ def _fill_from_environmental_variables(self):
                 env_variable_to_read = f"SIMTOOLS_{key.upper()}"
                 if value is None:
                     _env_dict[key] = os.environ.get(env_variable_to_read)
+                if key in _env_dict and _env_dict[key] is not None:
+                    _env_dict[key] = (
+                        _env_dict[key].split("#")[0].strip().replace(" ", "").replace("'", "")
```

Comment:
When would `replace(" ","")` be relevant (in an appropriate way)?
Do you think you need also `.replace('"', '')`?

Reply:
The environmental variable could be "SIMTOOLS_DB_API_PORT=27017_____#Port on the MongoDB server" (where '_" stands for spaces.

I haven't seen `"` anywhere, so I don't think we need them. Or should we add them to be on the safe side?

Reply:
But `strip` should take care of the extra spaces before `#`:
```
>>> a = "27017     # Port on the MongoDB server"
>>> a.split("#")[0].strip()
'27017'
```
Yes, I haven't seen any `"` anywhere either, but my suggestion was in order to be on the safe side.

Reply:
Agree